packages/cli/src/__tests__/commands/join.test.ts

```diff
@@ -64,6 +64,20 @@ describe('handleJoin fails', () => {
       },
       'cli-version'
     );
-    expect(writeYaml).toHaveBeenCalled();
+
+    expect(writeYaml).toHaveBeenCalledWith(expect.any(Object), 'openapi.yaml', expect.any(Boolean));
```

Comment:
Hi! Thanks for your contribution. 
Looks like this file doesn't pass the Prettier check - https://github.com/Redocly/redocly-cli/actions/runs/3851301286/jobs/6622465364
Could you please run Prettier on it?
Thanks.

Reply:
fixed